lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -99,31 +104,17 @@ final class AddComicViewController: UIViewController {
     }
 
     private func configureViewController() {
-        // Comic Title
-        configureTextFieldBorder(textField: comicTitleTextField)
-        comicTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Comic Title")
-        comicTitleTextField.addTarget(self, action: #selector(AddComicViewController.comicTitleEditingChanged), for: .editingChanged)
-
-        // Story Title
-        configureTextFieldBorder(textField: storyTitleTextField)
-        storyTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Story Title")
-
-        // Volume
-        configureTextFieldBorder(textField: volumeTextField)
-        volumeTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Volume Number")
-
-        // Issue
-        configureTextFieldBorder(textField: issueTextField)
-        issueTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Issue Number")
-
-        // Publisher
-        configureTextFieldBorder(textField: publisherTextField)
-        publisherTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Marvel, DC Comics, Image, Dark Horse…")
-
-        // Release Date
-        configureTextFieldBorder(textField: releaseDateTextField)
-        releaseDateTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Year of Release")
+
+        seriesTitleTextField.createBottomBorder()
```

Comment:
You don't need to call any of these `createBottomBorder` since that is done in the cell initialization now.